examples/standalone-task/main.tf

```
@@ -0,0 +1,120 @@
+provider "aws" {
+  region = local.region
+}
+
+data "aws_availability_zones" "available" {}
+
+locals {
+  region = "eu-west-1"
+  name   = "ex-${basename(path.cwd)}"
+
+  vpc_cidr = "10.0.0.0/16"
+  azs      = slice(data.aws_availability_zones.available.names, 0, 3)
+
+  container_name = "ecs-sample"
+  container_port = 80
+
+  tags = {
+    Name       = local.name
+    Example    = local.name
+    Repository = "https://github.com/terraform-aws-modules/terraform-aws-ecs"
+  }
+}
+
+################################################################################
+# Cluster
+################################################################################
+
+module "ecs_cluster" {
+  source = "../../modules/cluster"
+
+  cluster_name = local.name
+
+  # Capacity provider
+  default_capacity_provider_use_fargate = true
+  tags                                  = local.tags
+}
+
+################################################################################
+# A Mock service (for the sake of task definition only)
+################################################################################
+
+module "ecs_service" {
```

we can add this into the Fargate example instead of creating a new/standalone just for this

Lets change the name to better show what we're creating here
```suggestion
module "ecs_task_definition" {
```